src/bootstrap/bootstrap.py

```diff
@@ -463,10 +463,10 @@ def build_triple(self):
             cputype = 'i686'
         elif cputype in {'xscale', 'arm'}:
             cputype = 'arm'
-        elif cputype in {'armv6l', 'armv7l', 'armv8l'}:
+        elif cputype == 'armv6l':
             cputype = 'arm'
             ostype += 'eabihf'
-        elif cputype == 'armv7l':
+        elif cputype in {'armv7l', 'armv8l'}:
```

Comment:
I didn't mean that it was your device, just that it's the 64-bit Android I have at hand.  I think the difference is simply that JuiceSSH is a 32-bit app, and my Termux installation got 64-bit.